bouncer/blacklist/tests.py

```diff
@@ -33,6 +36,12 @@ def setUpClass(cls):
         )
         upper_case_email_entry.save()
 
+        user = User(username="example_user")
+        token = AuthToken(user=user)
+        cls.authenticated_client = APIClient()
+        cls.authenticated_client.force_authenticate(user=user, token=token)
```

Comment:
Please completely remove `user=user` from here and replace token with `token.key`. This will break tests and then you'll find the bug that exists 😇